### DISCUSSION TITLE
[RELEASE] Freddy Fazbear's Pizza

### DISCUSSION THREAD

**Discussion by panicmanic2410** [nzp-team/nzportable#340] [Map Releases]
Thought to toss my hat in the ring again with map making, but this time something more fun for me! Welcome to a night at Freddy Fazbear's Pizza!

![SS1](https://github.com/nzp-team/nzportable/assets/22651727/4d65841f-d833-4d42-af3f-2293f7da03ea)
![SS2](https://github.com/nzp-team/nzportable/assets/22651727/1130fb8f-b5a7-4106-b2da-821b898b45e7)
![SS3](https://github.com/nzp-team/nzportable/assets/22651727/dc4f5cae-5942-485e-a7b1-af649574476c)

**FEATURES:**
Pack-A-Punch, all Perk-A-Colas, Bouncing Betties, Electric Traps and custom models and audio!

### **DOWNLOAD**
[**PC**](https://github.com/nzp-team/nzportable/files/12528521/FNAF_PC.zip)
[**PSP**](https://github.com/nzp-team/nzportable/files/12528523/FNAF_PSP.zip)

Edited:

Alright, I've spent some time back at the drawing board and given this map another go. Rearranged its layout. The building itself remains mostly the same. I've only swapped around a few, but important objects! 

![GSS1](https://github.com/nzp-team/nzportable/assets/22651727/dea3e8b2-7de8-4d34-88dd-dece9fc5e39d)
The security room now has the power switch while the backstage has Pack-A-Punch instead. 

![GSS2](https://github.com/nzp-team/nzportable/assets/22651727/d91d4720-7832-40c4-9678-d390e8fcea11)
I've also touched up some of the interior's decor, now it feels like the proper abandoned pizzeria we all know and love!

Special thanks goes to blubs and MotoLegacy for helping me with the custom models.

As a small bonus...

[**Endoskeletons Zombie replacer!**](https://github.com/nzp-team/nzportable/files/12528565/Endo.zip)
For a truly immersive experience! Enjoy your time at Freddy Fazbear's Pizza!
(Remember to back up files and fair warning, their hitbox are very wonky! Aim for the head!)

**Comment by MotoLegacy** [comment]
You can get this booting on other platforms if you fix your models, all of your imported models have 256 duplicated frames, leading to that file size. And what happened to those external textures I made you?

EDIT: This should boot on everything but PSP though, so maybe revise your download button.

**Reply by panicmanic2410** [reply]
Hi Moto,

About the models... At the time including them, a part of me knew that these will be a tall order for this map and would obviously require optimization. This feedback has me reconsidering to revisit the models down the road, but right now I want to give it rest. 

As for the textures, Derp offered the suggestion and I was thankful for the help you did. The problem, I mistaken what was needed and sent the wrong textures. I thought they would be edited directly so I could use to export with again. I didn't know how you'll fix until it was sent and I realize it would be associated to each mdl. I separated the animatronics in parts to help load each of them. They each were given unique atlas textures since they originally had multiple materials. Take away, I messed up and was too embarrassed to ask you again to send me back with the corrected palettes.

For the port, I wanted to cut the custom models from mobile friendly version of the map. I don't have alot of know how on blender and optimization tricks. I would add them back eventually once I test the map to be playable on handheld. Apparently I was met with more problems trying. I thought cutting the custom models would be enough, but each revisions in my troubleshooting met with different errors until I've eventually given up. Maybe I'll look back at this one day. Didn't want the PC build of it to stay shelved while trying to fix the mobile one. Thanks for reading!

**Reply by MotoLegacy** [reply]
To be honest, I find it a little offensive that you deem exporting the models properly an optimization, when it's just fixing an export mistake. I did it myself in ~2 minutes, all you have to do is change the "End Frame: 250" playhead in blender to 1, so that you aren't duplicating your model two hundred and fifty times. The meshes themselves are fine, and now the full bodies are ~100kB each. It still is not booting on PSP 1000 because you aren't building the map fully (no vis calculation, which isn't an optimization, it's a very important part of the pipeline). Fixing that mistake will allow the Quake hunk to properly free things not in use per PVC. 

Here are those fixed meshes, with external textures. It shouldn't take you too long to fix your map, and I wish you wouldn't sorta loosely point at hardware as the issue, instead trying to identify mistakes made. All the best.
[Archive.zip](https://github.com/nzp-team/nzportable/files/12445520/Archive.zip)

**Comment by MotoLegacy** [comment]
"PC" version works well on PSVITA. Nice update.

**Comment by MotoLegacy** [comment]
![image](https://github.com/nzp-team/nzportable/assets/26030250/f14607e2-ee0e-4ebe-a167-34a26c3c6522)

Get rid of the massive box you have around the map though. It'll hurt performance everywhere.

**Comment by pahntomforces-guy** [comment]
i am so playing this

**Comment by jack5github** [comment]
This map is incredibly well made in terms of presentation, though there were a few issues I experienced during a play session with some of my friends. A few of the perk machines are completely non-solid, and the longer the game goes on, the more spawn points become glitched out. Essentially, zombies will stop entering into the map at certain spawn points, especially around the office area, and the number of spawn points this affects is significant and slows down the gameplay considerably, as we have to go hunting for the crawlers stuck underneath the visible entrances.

**Comment by Phynix0027** [comment]
The psp version is also compatible with 3ds?

**Comment by 26lake** [comment]
the ambience sfx gives me f*cking ptsd

**Comment by MotoLegacy** [comment]
Removed the "Meets Install Standard" tag as this doesn't follow the guidelines for download link visibility, and has versions specific to different platforms

**Comment by MiloVoid** [comment]
Does the PsP version work on 3DS or is there gonna be a separate one

**Comment by Phynix0027** [comment]
Hpw do I install the endoskeleton??

**Comment by diegops2pro-dot** [comment]
You Fall truth the floor on psp or at least the emulator

**Comment by patenaudejordan-creator** [comment]
im playing on the 3ds and i keep spawning beneath the map
